package.json

```diff
@@ -13,15 +13,45 @@
     "url": "https://github.com/Techtonica/curriculum/issues"
   },
   "scripts": {
-    "validate-links": "remark -u validate-links ."
```

Comment:
So…why was this removed? Why wouldn't we incorporate this into the pre-commit hook as well?